rl_loop.py

```diff
@@ -28,6 +28,7 @@
 BASE_DIR = "gs://{}".format(BUCKET_NAME)
 MODELS_DIR = os.path.join(BASE_DIR, 'models')
 SELFPLAY_DIR = os.path.join(BASE_DIR, 'games')
```

Comment:
While we're rejiggering our folder naming scheme, can we rename the selfplay dir? "games" is pretty nondescriptive.

Reply:
Done.